api/core/text2sql.py

```
@@ -441,10 +441,76 @@ async def generate():  # pylint: disable=too-many-locals,too-many-branches,too-m
+                            )
+                            
+                            if not healing_result.get("success"):
+                                # Healing failed after all attempts
+                                yield json.dumps({
+                                    "type": "healing_failed",
+                                    "final_response": False,
+                                    "message": f"❌ Failed to heal query after {healing_result['attempts']} attempt(s)",
+                                    "final_error": healing_result.get("final_error", str(exec_error)),
+                                    "healing_log": healing_result.get("healing_log", [])
+                                }) + MESSAGE_DELIMITER
+                                raise exec_error
```

Ruff (0.14.10)

Use `raise` without specifying exception name

Remove exception name

(TRY201)